UI/window-basic-main-outputs.cpp

```diff
@@ -565,7 +565,7 @@ void SimpleOutput::UpdateStreamingSettings_amd(obs_data_t *settings,
 	obs_data_set_int(settings, "FillerData", 1);
 	obs_data_set_int(settings, "VBVBuffer", 1);
 	obs_data_set_int(settings, "VBVBuffer.Size", bitrate);
-	
+
```

Comment:
That's trailing whitespace.  His editor probably removed it.  It doesn't really matter if trailing whitespace is removed, probably better that way.